subcommands/waves/show.go

```diff
@@ -0,0 +1,50 @@
+package waves
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	showCmd := &cobra.Command{
+		Use:   "show",
+		Short: "Show Ra given wave",
```

Comment:
"Show a given way"?

Reply:
OMG. this looks like a bad squash... it went really hard

Reply:
well no, it seems like you're looking at some old commit
the latest version has this correct

Reply:
np.